src/integration-test/resources/application-test.properties

```diff
@@ -4,6 +4,54 @@ logging.level.org.hibernate.SQL=off
 logging.level.org.ow2.proactive.catalog=info
 logging.level.org.springframework.web=info
 
+
+##############
+# DATASOURCE #
+##############
+
+######################## HSQL DB #######################################################
+# The default settings are using hsqldb
+spring.datasource.driverClassName=org.hsqldb.jdbc.JDBCDriver
+spring.datasource.url=jdbc:hsqldb:mem:catalog
+spring.jpa.database-platform=org.hibernate.dialect.HSQLDialect
+
+spring.datasource.username=root
+spring.datasource.password=
+
+######################## MYSQL DB #######################################################
```

Comment:
Maybe it's easier to use commented out configurations for various database, but why not use different files instead ?

Reply:
Using different files requires using the notion of 'profiles' in Spring. This implies to (i) hard-code the profile name just before the data source code and (ii) specify the profile at deployment time, which adds further useless constraints for the deployment of the catalog.  I prefer using a centralized properties file where we provide samples of different data sources. This is (as you already mentioned) easier for PA admin and Catalog tester.